packages/example/pages/_app.tsx

```diff
@@ -85,7 +87,12 @@ const RAINBOW_TERMS = 'https://rainbow.me/terms-of-use';
 const projectId =
   process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID ?? 'YOUR_PROJECT_ID';
 
-const chains = [
+const { wallets } = getDefaultWallets({
+  appName: 'RainbowKit demo',
```

Comment:
We drop `appName` param here. User will either call `getDefaultConfig` or `connectorsForWallets` and will need to supply the app metadata there

Reply:
Tbh i think we can drop this part. We already provide default wallets in `getDefaultConfig`. This would simplify the experience much more.